ambry-frontend/src/test/java/com.github.ambry.frontend/FrontendIntegrationTest.java

```diff
@@ -354,6 +385,30 @@ private void getBlobAndVerify(String blobId, HttpHeaders expectedHeaders, ByteBu
   }
 
   /**
+   * Gets the blob with blob ID {@code blobId} and verifies that the blob is not returned as blob is not modified
+   * @param blobId the blob ID of the blob to GET.
+   * @throws Exception
+   */
+  private void getNotModifiedBlobAndVerify(String blobId)
+      throws Exception {
+    HttpHeaders headers = new DefaultHttpHeaders();
+    SimpleDateFormat dateFormat = new SimpleDateFormat(RestUtils.HTTP_DATE_FORMAT, Locale.US);
```

Comment:
I think netty provides APIs to set date headers directly i.e. you can simply do 
```
headers.add(RestUtils.Headers.IF_MODIFIED_SINCE, new Date(System.currentTimeMillis()));
```
Don't think all this is required.